lib/Cake/Model/Datasource/Database/Mysql.php

```diff
@@ -895,8 +895,10 @@ public function insertMulti($table, $fields, $values) {
 				$i++;
 			}
 		}
+		$this->_connection->query('SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE="NO_AUTO_VALUE_ON_ZERO"');
```

Comment:
We can't keep this code here. It will triple query throughput to mysql for what is a very rare use case.